packages/mocks/src/getMockDataForRequest.ts

```diff
@@ -0,0 +1,67 @@
+import { PathParams } from 'msw';
+
+import * as providers from './data/provider.json';
+
+/**
+ * A mapping of API paths to their corresponding mock data.
+ */
+const mockData: Record<string, object> = {
+  '/api/proxy/plugin/forklift-console-plugin/forklift-inventory/providers': providers,
+};
+
+/**
+ * Get a mock response for a given request and path parameters.
+ *
+ * @param {Object} param0 - The object containing the request parameters
+ * @param {string} param0.pathname - The incoming request pathname to match against the mock handlers
+ * @param {string} param0.method - An optional string, the incoming request method to match against the mock handlers
+ * @param {PathParams} param0.params - An optional object containing the path parameters of the request
+ * @returns {MockResponse | null} A mock response or null if no mock handler is found
+ */
+export const getMockDataForRequest = ({
+  pathname,
+}: MockDataRequestParameters): MockResponse | null => {
+  // Add your mock handlers here, e.g.:
```

Comment:
@yaacov  have you tested that handlers will work outside of the MSW scope?

Reply:
yes, worked for me (tm),
can you also test ?